ui/components/app/assets/token-cell/cells/token-cell-secondary-display.tsx

```diff
@@ -80,6 +80,7 @@ export const TokenCellSecondaryDisplay = React.memo(
         ellipsis={token.isStakeable}
         isHidden={privacyMode}
         length={SensitiveTextLength.Medium}
+        style={{ whiteSpace: 'nowrap', paddingInlineStart: 8 }}
```

Comment:
I think we need to `useMemo` this because the new object in the past has triggered re-renders